documentation/starter-kits/GOVERNANCE/GOVERNANCE.md

```diff
@@ -0,0 +1,117 @@
+# [INSERT PROJECT NAME] Project Governance
+
+This governance model aims to create an open source community that encourages transparency, contributions, and collaboration, but maintains sound technical and quality standards. Our goal is to build a community comprised of members across the [INSERT PROJECT DOMAIN] community and beyond, including from private organizations, universities, government organizations, and international organizations. 
+
+The project follows a fairly liberal contribution model where people and/or organizations who do the most work will have the most influence on project direction. Technical decision making will primarily be made through a "[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)" approach. 
+
+## Roles
+
+### User
+
+Anyone who has downloaded, deployed, or operated [INSERT PROJECT NAME] to meet a specific objective. [INSERT PROJECT NAME] was primarily designed for [INSERT DESCRIPTION OF PROJECT PURPOSE], but let us know if you've found other uses for it.  
+
+### Contributor
+
+Contributors include anyone that provides input to the project. This includes code, issues, documentation, graphics, designs, or anything else that tangibly improves the project. We encourage you to start contributing right away by joining our [Discussions]([INSERT LINK TO DISCUSSION BOARD]) or submitting an [Issue]([INSERT LINK TO ISSUE TRACKING SYSTEM]). 
+
+### Triager
+
+Subset of contributors who have demonstrated a strong familiarity with [INSERT PROJECT NAME] and are regularly contributing to the project via issue creation, commenting, discussions, etc. Triagers are given the "Triage" GitHub role and are added to corresponding GitHub team. These team members have the ability to do the following:
+
+    - Label issues and pull requests
+    - Comment, close, and reopen issues and pull requests
+
+[List of current Triagers]([INSERT LINK TO GITHUB TEAM ASSOCIATED WITH TRIAGERS])
+ 
+### Collaborator

```

Comment:
Curious - what made you choose the term "Collaborator" as opposed to "Committer"?